source/Calamari.Aws/Deployment/CloudFormation/StackStatus.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Calamari.Aws.Deployment.CloudFormation
```

Comment:
How were these values represented before?

Reply:
Was nested in the `DeployAwsCloudFormationConvention.cs`